src/snowflake/cli/plugins/object/stage/diff.py

```diff
@@ -154,12 +159,31 @@ def build_md5_map(list_stage_cursor: SnowflakeCursor) -> Dict[str, str]:
     }
 
 
-def stage_diff(local_path: Path, stage_fqn: str) -> DiffResult:
+def get_absolute_files_to_stage(relative_files_to_stage, local_path) -> List[Path]:
```

Comment:
nit: _ prefix? It seems rather specific to local_path == deploy root, so I'd consider it an private API within this module.